bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/internal/WindowsDefenderConfigurator.java

```diff
@@ -202,6 +204,9 @@ protected Control createCustomArea(Composite parent) {
 							}).create(parent);
 					GridDataFactory.swtDefaults().indent(0, 5).applyTo(keepScanning);
 
+					if (!PlatformUI.isWorkbenchRunning()) {
```

Comment:
A custom app can still use the e3 compatibility later and then there will be a workbench running... so the comment is quite misleading.

Reply:
Right, corrected that comment.